boot/environ/include/efi/UgaDraw.h

```diff
@@ -85,7 +85,7 @@ typedef union {
 } EFI_UGA_PIXEL_UNION;
 
 ///
-/// Enumration value for actions of Blt operations.
+/// Enumeration value for actions of Blt operations.
```

Comment:
Note that all these headers under include/efi are 3rd-party, so these changes should be either reverted, or somehow sent upstream...